src/debug/debug.cpp

```diff
@@ -2354,11 +2369,14 @@ bool CDebugVar::LoadVars(char *name)
 }
 
 static void SaveMemory(uint16_t seg, uint32_t ofs1, uint32_t num) {
-	FILE* f = fopen("MEMDUMP.TXT","wt");
+	const std::string memdump_txt = "MEMDUMP.TXT";
+	FILE* f = fopen(memdump_txt.c_str(),"wt");
```

Comment:
Change `memdump_txt` to `std_fs::path`; you method `c_str()` is implemented there as well (ditto for other file names in this commit).

Reply:
resolved by directly using .string().c_str()

Reply:
But you haven't changed the type of `memdump_txt` as I asked to… `MEMDUMP.TXT` is a relative path after all.